src/ValueToggle/ValueToggle.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from 'styled-components'
+
+import { Color, theme } from '../theme'
+import { Box } from '../Box'
+import { Tag } from '../Tag'
+import { Text } from '../Text'
+import React from 'react'
+import { ToggleButton } from './ToggleButton'
+
+export type ValueToggleOption<T> = {
+  label: string
+  value: T
+  tag?: string // optional tag able to show in the toggle if showTag prop is passed
```

Comment:
Can we turn all of these comments into JSDocs? this would ensure IDEs are able to pick them up when people are inspecting the properties e.g.
```suggestion
  /**
   * optional tag able to show in the toggle if showTag prop is passed
   */
  showTag?: boolean 
```